ci fails: Test Leaderboard Dockerfile

Seems like this CI has failed for a long time. 

E.g. here: 

https://github.com/embeddings-benchmark/mteb/pull/4108

where it fails with:

```
2026-02-23 09:03:56,350 - PID:12 - httpx - INFO - HTTP Request: GET https://api.gradio.app/pkg-version "HTTP/1.1 200 OK"
2026-02-23 09:04:00,449 - PID:12 - __main__ - INFO - Step 7/7 complete: Built Gradio interface in 4.63s
2026-02-23 09:04:00,449 - PID:12 - __main__ - INFO - Starting prerun on all benchmarks to populate caches...
mteb-test
mteb-test
Error: Process completed with exit code 1.
```

I am not sure why it fails, though

--- a/.github/workflows/leaderboard_docker.yml
+++ b/.github/workflows/leaderboard_docker.yml
@@ -72,7 +72,7 @@ jobs:
         docker tag ghcr.io/${{ github.repository }}/leaderboard:${{ github.sha }} mteb-leaderboard:test
 
     - name: Test container can start and run
-      timeout-minutes: 9
+      timeout-minutes: 12
       run: |
         # Start the container in background
         docker run -d --name mteb-test -p 7860:7860 mteb-leaderboard:test
@@ -88,8 +88,8 @@ jobs:
           CURRENT_TIME=$(date +%s)
           ELAPSED=$((CURRENT_TIME - START_TIME))
 
-          # Maximum timeout: 8 minutes - always fail at timeout
-          if [ $ELAPSED -gt 480 ]; then
+          # Maximum timeout: 12 minutes - always fail at timeout
+          if [ $ELAPSED -gt 720 ]; then
             echo "❌ Timeout reached after ${ELAPSED}s - container failed to complete initialization"
             docker logs --tail 20 mteb-test
             docker stop mteb-test

